reference/tax.v3.yml

```diff
@@ -207,4 +207,20 @@ components:
                 Describes whether the stored credentials are considered complete and configured, ready to be used for Tax Provider API requests.
 
                 Merchants may enable any **configured** tax provider for storefront tax quotation.
+            target:
+              type: object
+              description: A checkout session's active tax provider connection will be determined by the target data.
+              properties:
+                country_codes:
+                  type: array
+                  items:
+                    type: string
+                  description: The list of country codes where the tax provider connection is active. ISO 3166-1 alpha-2 country code.
+                  example: ['AU', 'US']
+                subdivision_codes:
+                  type: array
+                  items:
+                    type: string
+                  description: The list of subdivision codes where the tax provider connection is active. ISO 3166-2 subdivision code, up to three alphanumeric characters, each with a ISO 3166-1 alpha-2 prefix.
```

Comment:
The description was a bit confusing around the `ISO-3166-2 subdivision code, up to three alphanumeric characters...`

What do you think about starting with describing the country prefix then describing the subdivision bit after?

Reply:
Good idea, it's out of order as it is now. Thanks!

Reply:
Actually, turns out this added description isn't needed - it's all part of the ISO 3166-2 standard.

That is `AU-NSW` is a complete ISO 3166-2 code, per the standard. I'll simplify the language.